var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -27,138 +27,531 @@
-            description='Enable MPI parallelism')
-    variant('lapack',  default=True,
-            description='Use external BLAS/LAPACK libraries')
-    variant('klu',     default=False,
+            description='Enable MPI parallel vector')
+    variant('openmp',  default=False,
+            description='Enable OpenMP parallel vector')
+    variant('pthread', default=False,
+            description='Enable Pthreads parallel vector')
+    variant('cuda',    default=False,
+            description='Enable CUDA parallel vector')
+    variant('raja',    default=False,
+            description='Enable RAJA parallel vector')
+
+    # External libraries
+    variant('blas',       default=False,
+            description='Enable external BLAS libraries')
+    variant('lapack',     default=False,
+            description='Enable external LAPACK libraries')
+    variant('klu',        default=False,
             description='Enable KLU sparse, direct solver')
-    variant('superlu', default=False,
+    variant('superlu-mt', default=False,
             description='Enable SuperLU_MT sparse, direct solver')
-    variant('openmp',  default=False,
-            description='Enable OpenMP parallelism')
-    variant('pthread', default=True,
-            description='Enable Pthreads parallelism')
-    variant('hypre', default=False,
-            description='Enable Hypre parallel vector for MPI parallelism')
-
-    depends_on('cmake',              type='build')
-    depends_on('mpi',                when='+mpi')
-    depends_on('mpi',                when='@2.7:+hypre')
-    depends_on('hypre',              when='@2.7:+hypre')
-    depends_on('blas',               when='+lapack')
+    variant('hypre',      default=False,
+            description='Enable Hypre MPI parallel vector')
+    variant('petsc',      default=False,
+            description='Enable PETSc MPI parallel vector')
+
+    # Library type
+    variant('shared',  default=True,
+            description='Build shared libraries')
+    variant('static',  default=True,
+            description='Build static libraries')
+
+    # Fortran interface
+    variant('fcmix', default=False,
+            description='Enable Fortran interface')
+
+    # Examples
+    variant('examples-c', default=True,
+            description='Enable C examples')
+    variant('examples-cxx', default=False,
+            description='Enable C++ examples')
+    variant('examples-f77', default=True,
+            description='Enable Fortran 77 examples')
+    variant('examples-f90', default=False,
+            description='Enable Fortran 90 examples')
+    variant('examples-cuda', default=False,
+            description='Enable CUDA examples')
+    variant('examples-raja', default=False,
+            description='Enable RAJA examples')
+    variant('examples-install', default=True,
+            description='Install examples')
+
+    # Generic (std-c) math libraries (UNIX only)
+    variant('generic-math', default=True,
```

Comment:
does it work on macOS? if not, maybe set `default = platfom!="darwin"` or alike.

Reply:
Our CMake build links to std-c math libraries only if 'IF(UNIX)' is true. This option is really for a unix user to turn this behavior off, otherwise it's not used.

Reply:
ah, ok. Just wanted to be sure that it won't break the build on macOS with default values (the current version of the package builds OK).